website/layouts/partials/_default/sidemenu.html

```diff
@@ -1,68 +1,149 @@
 {{ $currentPage := . }}
+{{- $currentVersion := index (split .File.Path "/") 1 }}
+{{- $versionDocsPath := printf "docs-pre-processed/%s" $currentVersion }}
+{{ $url := .RelPermalink }}
 
 <button class="btn btn-block btn-outline-secondary mb-3 d-block d-lg-none" type="button" data-toggle="collapse" data-target=".docs-menu">
     Toggle Menu
 </button>
 
-<nav class="docs-menu collapse d-lg-block">
+{{ range .Site.Sections }}
+
+{{ $docsPage := . }}
+{{ $urll := .RelPermalink}}
+ 
+<nav class="docs-menu collapse d-lg-block"> 
+
+  {{ range .Pages.GroupBy "Section" }}
+  {{ if in $url $urll }}
```

Comment:
`url` is same as `urll` so it's always true

Reply:
Always true 😉👍